requirements/prod.in

```diff
@@ -12,21 +10,16 @@ django-hashid-field==3.1.1
 django-js-reverse==0.9.1
 django-log-request-id==1.4.1
 django-model-utils==4.0.0
-django-parler==2.0.1
 django-redis==4.11.0
 django-rq==2.3.0
-django-storages==1.9.1
 djangorestframework==3.11.0
 furl==2.1.0
 github3-py==1.3.0
 rq-scheduler==0.9.1
 logfmt==0.4
 markdown==3.2.1
-newrelic==5.8.0.136
 psycopg2-binary==2.8.4
-pyOpenSSL==19.1.0
 sentry-sdk==0.14.2
-service_identity==18.1.0
```

Comment:
While building, I now get this error/warning:

```
Step 21/22 : RUN python manage.py collectstatic --noinput
 ---> Running in 9124cc2a254c
:0: UserWarning: You do not have a working installation of the service_identity module: 'No module named 'service_identity''.  Please install it from <https://pypi.python.org/pypi/service_identity> and make sure all of its dependencies are satisfied.  Without the service_identity module, Twisted can perform only rudimentary TLS client hostname verification.  Many valid certificate/hostname mappings may be rejected.
```

I think `channels` requires `daphne`, which requires `twisted[tls]`, which requires `pyopenssl`, `service_identity`, and `idna`. The other two are already installed along with `requests`, but now `service_identity` is missing.

https://github.com/twisted/twisted/blob/trunk/src/twisted/python/_setup.py#L96-L104

Reply:
Hmm, okay, I'll revisit. The fact that I was able to run the tests without seeing this warning suggests that our Python tests do not make sure that we can run daphne.

Reply:
That's to be expected; the tests are about our logic, not about our ASGI server. But I was able to run it fine to hand-test, as far as I could see, too, which surprises me given that error.